### ISSUE TITLE
Cloudflare sites cause ---Timeout of 48000ms exceeded

### ISSUE THREAD

**Issue by WesNz** [louislam/uptime-kuma#4764]
### ⚠️ Please verify that this question has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Websites using cloudflare for dns are causing lots of "Timeout of 48000ms exceeded" errors
: Failing: timeout of 48000ms exceeded | Interval: 60 seconds | Type: http | Down Count: 0 | Resend Interval: 0

I have followed the side note ( https://github.com/louislam/uptime-kuma-wiki/blob/master/Cloudflare-Side-Note.md )
Added the IP to Access rules. 
Disabled "Browser Integrity Check"

### 🐻 Uptime-Kuma Version

 Version: 1.23.13 Frontend Version: 1.23.13

### 💻 Operating System and Arch

Docker

### 🌐 Browser

Brave

### 🖥️ Deployment Environment

- Runtime: 
- Database: Yes
- Filesystem used to store the database on: 
- number of monitors:

**Comment by CommanderStorm** [comment]
@pieteru771 entirely different issue. Please search through the existing issues https://github.com/louislam/uptime-kuma/issues?q=is%3Aissue+EAI_AGAIN

TL;DR: Either your IP-config is borked, your DNS server is overwhealmed (enabled NSCD in the settings?) or a third issue. None of them are timeouts ^^

**Comment by CommanderStorm** [comment]
about the original timeout issue:
Could you try to reproduce this issue via curl and report what the response time is?
https://github.com/louislam/uptime-kuma/wiki/Troubleshooting

Also: is this consistently failing or just intermitently? (in the later case, this is cloudflares problem or networkign schenangans and you should increase the retreies from 0)

**Comment by WesNz** [comment]
It a basic html site.
Randomly fails often. I increased retires to 5....is that too low?
Down | 2024-05-17 05:25:46 | timeout by AbortSignal (48s)


2024-05-17T18:48:18+12:00 [MONITOR] WARN: Monitor #6 'Websites': Pending: Child inaccessible | Max retries: 5 | Retry: 0 | Retry Interval: 60 seconds | Type: group
2024-05-17T19:10:28+12:00 [MONITOR] WARN: Monitor #20 'Aspergers.co.nz': Pending: timeout by AbortSignal (48s) | Max retries: 5 | Retry: 1 | Retry Interval: 60 seconds | Type: http
2024-05-17T19:11:19+12:00 [MONITOR] WARN: Monitor #6 'Websites': Pending: Child inaccessible | Max retries: 5 | Retry: 0 | Retry Interval: 60 seconds | Type: group


</body>
</html>

root@uptime:/app# curl https://aspergers.co.nz
<HTML>
<HEAD>
<TITLE>Jen Birch - Congratulations! It▒s Asperger Syndrome</TITLE>
<meta name='Keywords' content="Asperger Syndrome, Autistic Spectrum Disorder, Autism, Jen Birch, Congratulations! It▒s Asperger Syndrome, Developmental Disorder, Social impairment, Diagnostic Criteria, book education, New Zealand">
<meta name='Description' content="This website seeks to give basic information on Autistic Spectrum Disorder, with an emphasis on the higher-functioning types of Asperger▒s Syndrome and High-Functioning Autism. Checklists are included. To assist interested persons, I have also written a book on Asperger▒s Syndrome.">

<LINK rel="stylesheet" type="text/css" href="style.css">
<link rel="STYLESHEET" type="text/css" href="cm/cm.css">
<script language="JavaScript1.2" src="cm/coolmenus4.js">
/* Copyright (c) 2001 Thomas Brattli (webmaster@dhtmlcentral.com)
DHTML coolMenus - Get it at coolmenus.dhtmlcentral.com
Version 4.0_beta */
</script>
<STYLE TYPE="text/css">@import url('cm/ns4fix.css');</STYLE>
<script language="Javascript" type="text/javascript" src="site.js"></script>
</HEAD>
<BODY class="home"><script type="text/javascript" language="JavaScript" src="cm/cm.js"></script>
<center>
<div align=center>
<table border="0" cellspacing="0" cellpadding=0 width="760px">
  <tr>
    <td width="100%" class="banner">
        <table border="0" cellspacing="0" cellpadding=0 width="100%">
          <tr>
            <td class="publisher"><A HREF="http://www.jkp.com/" target="_blank"><IMG SRC="images/publisher.gif" ALT="Jessica Kingsley Publishers" BORDER="0" WIDTH="89px" HEIGHT="100px"></A></td>
            <td class="title"><img border="0" src="images/title.gif"></td>
            <td class="pic"><img border="0" src="images/bannerPic.jpg"></td>
          </tr>
        </table>
  </tr>
  <tr>
    <td width="100%" class="navContainer">&nbsp;</td>
  </tr>
  <tr>
    <td class="content"><p>It is now recognised that autism occurs in a variety of forms, e.g.
High-Functioning Autism and Asperger▒s Syndrome (sometimes called Asperger▒s
Disorder). Autism and related conditions can now be grouped under the term
Autistic Spectrum Disorder (A.S.D.) They are relatively common, with Asperger▒s
Syndrome occurring in one in every 300 persons.</p>
<p>This website seeks to give basic information on Autistic Spectrum Disorder,
with an emphasis on the higher-functioning types of Asperger▒s Syndrome and
High-Functioning Autism. Checklists are included. To assist interested persons,
I have also written a book on Asperger▒s Syndrome, (see book page). </p>
<p>&nbsp;</p>
</td>
  </tr>
  <tr>
    <td class="footer"><p>I can be contacted on Email: <a href="mailto:jenbirch@actrix.gen.nz">
jenbirch@actrix.gen.nz</a> <br>
or by post: Jen Birch, PO Box 17-426, Greenlane, Auckland, New Zealand. <br>
I welcome messages and enquiries.</p>
<p class=links><a href="index.shtml">Home</a> |
<a href="whatIsAsperger.shtml">What is Asperger Syndrome?</a> |
<a href="autismDevDisorder.shtml">Autism is a Developmental Disorder</a> |
<a href="insightAutistic.shtml">Insight into an Autistic Spectrum Disorder</a><br>
<a href="recReadings.shtml">Recommended Readings</a> |
<a href="whereToObtain.shtml">Where to Obtain this Book</a> |
<a href="contact.shtml">Contact Jen</a> | <a href="links.shtml">Links</a></p></td>
  </tr>
</table>
</div>
</center>
</BODY>
</HTML>root@uptime:/app# ping aspergers.co.nz
PING aspergers.co.nz (45.77.232.238) 56(84) bytes of data.
64 bytes from 45.77.232.238.vultrusercontent.com (45.77.232.238): icmp_seq=1 ttl=45 time=318 ms
64 bytes from 45.77.232.238.vultrusercontent.com (45.77.232.238): icmp_seq=2 ttl=45 time=320 ms
64 bytes from 45.77.232.238.vultrusercontent.com (45.77.232.238): icmp_seq=3 ttl=45 time=321 ms
64 bytes from 45.77.232.238.vultrusercontent.com (45.77.232.238): icmp_seq=4 ttl=45 time=313 ms
64 bytes from 45.77.232.238.vultrusercontent.com (45.77.232.238): icmp_seq=5 ttl=45 time=322 ms
^C
--- aspergers.co.nz ping statistics ---
6 packets transmitted, 5 received, 16.6667% packet loss, time 5ms
rtt min/avg/max/mdev = 312.758/318.881/322.416/3.534 ms
root@uptime:/app#

**Comment by CommanderStorm** [comment]
Can't reproduce the issue on demo.kuma.pet
![image](https://github.com/louislam/uptime-kuma/assets/26258709/1df740b9-58e6-450c-8d69-ce081938f053)

I think it might be relateed to either
- cloudflare dermining that this request should be blocked
- wrong network configuration

Given that basic curl works, I think it might be the first.
=> Could you report a hunch i have from inside your contianer?
```bash
time curl -v https://aspergers.co.nz
```

=> For Cloudflare could you please double-check that there has not been a typo somewhere by re-copy pasing all fields?

**Comment by sw7ft** [comment]
Hey we verified this, we have monitors running on different servers and were able to confirm that Cloudflare blocked on our monitor  and because we run 45-second timeout on our requests we saw a lot of them in one day

<img width="278" alt="Screenshot 2024-07-10 at 2 36 58 PM" src="https://github.com/louislam/uptime-kuma/assets/149521204/40522f73-71b4-45dd-8077-7bccdbc0ceab">

we have some domains on one monitor that are using cloudflare and others that are not, the ones all using cloudflare were banned to access the above cloudflare ip

the ban lasted roughly 5mins

**Comment by zilch-gg** [comment]
Having the same issue here. It only happens when using Cloudflare Proxy.
![image](https://github.com/user-attachments/assets/37524754-679a-4ee9-a02a-5f2cc99f5348)

**Comment by jordantrizz** [comment]
> we have some domains on one monitor that are using cloudflare and others that are not, the ones all using cloudflare were banned to access the above cloudflare ip

How'd you confirm the "ban" you should have logs under Security->Events?

**Comment by layerly** [comment]
I am also having the exact same issues with the "timeout by AbortSignal (48s)" on multiple sites. This happens to the same sites almost every day. When I check the site, it appears to be up and running properly without any issues.

Also using Uptime Kuma and Cloudflare. This probably started in July for me.

<img width="751" alt="Screenshot 2024-08-29 at 8 10 15 AM" src="https://github.com/user-attachments/assets/55b95f15-a1ca-4fb4-a9a5-a2195c9fda8a">

**Comment by deadlocks** [comment]
Facing same issue, checked CF, there are no logs under security

**Comment by bingloo** [comment]
+1, Facing same issue, checked CF, there are no logs under security

**Comment by GordonHannan** [comment]
I also was seeing this with a number if sites with the 1.23.15 release. I just upgraded to 2.0 beta so I'll see if it keeps happening there too.

**Comment by Sneeex** [comment]
+1, no logs under security in CF
